src/ConnectedMode.UnitTests/Binding/BindingProcessImplTests.cs

```diff
@@ -161,14 +162,13 @@ private static ServerExclusions CreateSettings()
         [TestMethod]
         public async Task DownloadQualityProfile_SavesConfiguration()
         {
-            var configPersister = new ConfigurableConfigurationProvider();
+            var configPersister = new DummyConfigPersister();
 
             // Arrange
             const string QualityProfileName = "SQQualityProfileName";
             const string ProjectName = "SQProjectName";
 
-            var notifications = new ConfigurableProgressStepExecutionEvents();
-            var progressAdapter = new FixedStepsProgressAdapter(notifications);
```

Comment:
We don't need to use the real adapter class in the test (and don't want to, since it is in the `Integration` assembly which we don't want to reference from the `ConnectedMode.UnitTests`)